src/shared/ui/survey/SimpleTextInput.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC, useState } from 'react';
+import { TextInputProps } from 'react-native';
+
+import { useTranslation } from 'react-i18next';
+
+import { colors } from '@shared/lib';
+import { Input } from '@shared/ui';
+
+type Props = {
+  config: {
+    maxLength: number;
+    valueType: '' | 'http://www.w3.org/2001/XMLSchema#integer';
```

Comment:
Why should we relay on schema definitions in Mobile app?
The type might be described with simple Enum value